vip-cache-manager/vip-cache-manager.php

```diff
@@ -4,16 +4,25 @@
 Description: Automatically clears the Varnish cache when necessary
 Author: Automattic
 Author URI: http://automattic.com/
-Version: 1.0
+Version: 1.1
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+require_once( __DIR__ . '/api.php' );
+
 class WPCOM_VIP_Cache_Manager {
 	private $ban_urls = array();
 	private $purge_urls = array();
 	private $site_cache_purged = false;
 
-	function __construct() {
+	static public function instance() {
+		static $instance = false;
+		if ( ! $instance )
+			$instance = new WPCOM_VIP_Cache_Manager;
```

Comment:
Can we use the WP coding standard `{}` braces, and throw a `()` on the instantiation?